src/com/theironyard/novauc/CInput.java

```diff
@@ -0,0 +1,55 @@
+package com.theironyard.novauc;
+
+import java.util.Scanner;
+
+
+class CInputProgram {
+
+    public static Scanner scannerln = new Scanner(System.in);
+    public static String input = scannerln.nextLine();
+    public static int intInput = scannerln.nextInt();
+    public static Comedian comedian = new Comedian();
+
+    public static void main(String[] args) throws Exception {
+
+        System.out.println ("Hi! What would you like to know about our Comedian?");
+
```

Comment:
D'Angelo, I don't think you are clear on what should be happening.  Come talk to me about this assignment when you get a chance so we can get you back on track.